### DISCUSSION TITLE
How can I find out how much bandwidth I've used?

### DISCUSSION THREAD

**Discussion by projeffboy** [vercel/vercel#6166] [Help]
I remember when the site was called ZEIT Now, I was able to track my bandwidth. Now it seems to be a paid feature. But how do I know if I've exceeded the 100 GB limit? My website is starting to get 600 users a day, and I don't want to have my site closed suddenly because I've exceeded the free limit (see [this person's post](https://github.com/vercel/vercel/discussions/5979))

**Comment by mcsdevv** [comment]
Hey @projeffboy, all Pro plans include access to a usage dashboard!
